sn_node/examples/registers.rs

```diff
@@ -63,9 +63,11 @@ async fn main() -> Result<()> {
 
     // we'll retrieve (or create if not found) a Register, and write on it
     // in offline mode, syncing with the network periodically.
+
+
     let meta = XorName::from_content(reg_nickname.as_bytes());
     let address = RegisterAddress::new(meta, client.signer_pk());
-    println!("Retrieving Register '{reg_nickname}' from SAFE, as user '{user}'");
+    println!("Retrieving Register '{reg_nickname}':'{address:?}' from SAFE, as user '{user}'");
```

Comment:
Is it possible if we can call `address.to_hex()` while we're displaying the address to the user? 

All the different address types inc. `RegisterAddress` were previously truncated to display the first couple chars to not fill up the log files.

Reply:
Is logging the first couple chars enough for folk to follow the logs for debugging? It wasn't for me, but it's very possible I just don't fully understand. 

Also, @dirvine mentioned [here](https://safenetforum.org/t/communication-via-safe-network/38958/14?u=trycatch) it's a bug:

> Looks like log output was cleaned up and the whole id (64byte) truncated for easier reading. In this case that is wrong and it should be printed out clearly for you to read.
> Bug bug !!

Ultimately though, I leave it up to you core devs if you want to incorporate this change or not, you have a much bigger picture of things to consider.

Reply:
Yes, that makes sense.  I update to log the entire RegisterAddress only at few key places.